backend/certificates.js

```diff
@@ -34,3 +34,28 @@ export function saveCertsLocally(json) {
     saveSetting('CONNECTOR_HOST_INFO', hostInfo);
     return {};
 }
+export const msInOneDay = 1000 * 3600 * 24;
+
+export function calculateDaysToRenewal() {
+    const daysCertificateIsGoodFor = 24; // Because max setTimout value is 24 days.
+    const {lastUpdated} = getSetting('CONNECTOR_HOST_INFO');
+    const timePassed = Math.abs(new Date().getTime() - new Date(lastUpdated).getTime());
```

Comment:
why `Math.abs`? I'm generally not in favor of these extra safety checks because they can obscure a bigger underlying problem. For example, if this returns a negative time, we have a bigger and more unknown problem on our hands that we should be aware about as soon as possible